### ISSUE TITLE
🐧 The Aquatic Adventure of the Last Human ↔ v18.6.0

### ISSUE THREAD

**Issue by ClementRivaille** [itchio/itch-compatibility-watchlist#64]
:rotating_light: game [The Aquatic Adventure of the Last Human](https://ycjy.itch.io/aquaticadventure) is broken for me.

:book: Here's the complete [debug log](https://gist.github.com/f16fb6d5050e152d27165ee27fad2a87).

:running: Any additional details can go here!

**Comment by fasterthanlime** [comment]
Are you seeing anything that looks like a Linux executable in the demo archive? It might just be a case of overzealous tagging by the game developer.

**Comment by ClementRivaille** [comment]
The demo doesn't have any indeed. But the full game does. It is a .zip that contains three .rar (sorry), one for each platform.
The Linux one seems confusing though. It has a _run.sh_ file that only executes a _"The Aquatic Adventure of the Last Human" binary. Except that there is no such binary, and instead there is a _"TheAquaticAdventureOfTheLastHuman"_ one.
However I can't confirm if this is voluntary or not. After editing the .sh, I had issues with a missing library. But that's another unrelated story.

On an other note, I then tried to run it on Windows. The game couldn't be launched from itch first, but it works now that I have manually extracted the _windows.rar_ archive.
Should this be on another issue?

**Comment by fasterthanlime** [comment]
Honestly, just pointing the developer at https://itch.io/docs/itch/integrating/quickstart.html would probably go a long way towards fixing all of that.

TL;DR:
- .rar is supported now, but please don't use it for anything new
- one upload per platform is what the app (and most folks) expect
- just use [butler](https://itch.io/docs/butler), or .zip for everything, or .7z if you must
- test your linux launcher scripts
